docs/policy/community-testing.md

```diff
@@ -29,20 +29,22 @@ After users have been notified of changes, minor packages will be marked eligibl
 negative feedback after 7 calendar days.
 In addition to the above requirements, major packages must also receive positive feedback and be approved by the Release
 Manager.
+If a major package's functionality is tested by the OSG VM Universe tests, the release manager may approve that
+package for release without positive feedback after the package has been in testing for four weeks.
 If a package receives negative feedback, the offending package will be removed from the relevant testing repository.
 
 Major Packages
 --------------
 
 The following packages are considered critical to the production Open Science Grid:
 
-- BLAHP
```

Comment:
Should we keep this in since it's still a separate package in 3.6 release?

Reply:
I can go either way on this. We don't plan any more blahp releases and the integrated blahp is in wide used.

Reply:
Restored